gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt

```diff
@@ -22,7 +22,7 @@ internal object ComposeCompilerCompatibility {
         "1.9.20-RC2" to "1.5.3-rc01",
         "1.9.20" to "1.5.3",
         "1.9.21" to "1.5.4",
-        "1.9.22" to "1.5.7.1",
+        "1.9.22" to "1.5.8-beta01",
```

Comment:
Let's create a task "Release Compose Compiler 1.5.8" and add "required for" link for "Release Compose Multiplatform 1.6"